### PR TITLE
mysqldef: fix TestCodec location bug.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
fix issue #68.

### CODE REVIEW

File: mysqldef/time_test.go [hunk 1]
```diff
@@ -289,6 +289,26 @@ func (s *testTimeSuite) TestYear(c *C) {
 
 }
 
+func (s *testTimeSuite) getLocation(c *C) *time.Location {
+	locations := []string{"Asia/Shanghai", "Europe/Berlin"}
+	timeFormat := "Jan 2, 2006 at 3:04pm (MST)"
```
Comment (siddontang) [inline mysqldef/time_test.go]: why not use TimeFormat directly ?

**Comment by shenli** [comment]
LGTM

File: mysqldef/time_test.go (hunk 1 above)
Comment (qiuyesuifeng) [inline mysqldef/time_test.go]: timeFormat and value variable is from golang.org/time ParseInLocation example.
i think here it is no need to use TimeFormat, for it is type of  `2006-01-02 15:04:05`, i will have to modify value variable.

Comment (qiuyesuifeng) [inline mysqldef/time_test.go]: perhaps u are right, i will use TimeFormat to update this test case.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**